### PR TITLE
fix: MYNW-114 Temporary hide bridge section from topup-page

### PR DESCRIPTION
**Pull request by followdarko**
MYNW team decided to hide bridge section from Topup page because the only one option here is Rainbow Bridge. Right now MYNW users can't use Rainbow Bridge because of lack of integration. Work in progress on this issue, but for now best option is not to confuse users and hide section.

### CODE REVIEW

File: packages/frontend/src/components/buy/BuyNear.js [hunk 1]
```diff
@@ -225,11 +226,13 @@ export function BuyNear({ match, location, history }) {
                     subTitle='buyNear.nearPurchaseSubTitle'
                     actions={[PayMethods.nearPay, PayMethods.moonPay, PayMethods.utorg, PayMethods.ftx]}
                 />
-                <FundingCard
-                    title='buyNear.bridgeTokens'
-                    subTitle='buyNear.bridgeSubTitle'
-                    actions={[PayMethods.rainbow]}
-                />
+                {!isWhitelabel && (
```
Comment (Patrick1904) [inline packages/frontend/src/components/buy/BuyNear.js:229]:
```suggestion
                {!isWhitelabel() && (
```

**Review by Patrick1904 — CHANGES_REQUESTED** [review]
LGTM - just make sure to run the function

File: packages/frontend/src/components/buy/BuyNear.js (hunk 1 above)
Comment (Patrick1904) [reply packages/frontend/src/components/buy/BuyNear.js:229]: Not a blocker: may be worth to make `isWhiteLabel` a selector vs running this manually every time

Comment (followdarko) [reply packages/frontend/src/components/buy/BuyNear.js:229]: Oh! Thank you! Always forgot that this is a function. Actually naming like boolean variable really confusing. If we go deeper then why does it's even a function while it behaves like a real boolean flag!? No need to make any selector I refactor code to use it like a variable.

**Comment by followdarko** [comment]
@Patrick1904 Sorry, I'm refactored a little bit. Hope you don't mind

**Comment by Patrick1904** [comment]
> @Patrick1904 Sorry, I'm refactored a little bit. Hope you don't mind

Thanks @followdarko. Can you please just rebase this PR and make sure all usage of `isWhiteLabel` is correct after this change (some usage in latest [release](https://github.com/near/near-wallet/pull/2869)), and we'll this merged ASAP.

**Comment by followdarko** [comment]
Outdated since Rainbow Bridge integrate support for MyNearWallet.

**Pull request closed without merging**